packages/web-react/src/components/CheckboxField/CheckboxField.tsx

```diff
@@ -1,6 +1,6 @@
 import classNames from 'classnames';
 import React from 'react';
-import { useStyleProps } from '../../hooks/styleProps';
+import { useStyleProps, useDeprecationMessage } from '../../hooks';
```

Comment:
Do we sort these alphabetically?

Reply:
Fixed, but I usually give it as it is called. So `useStyle` is called first and then `useDeprecation`. 
As a result, it has no effect at all.